README.md

```diff
@@ -40,9 +53,10 @@ Default path to config file is
   ```
 
 ## Driver Installation
-After the above Prerequisites are met, Use the standard golang install process:
+After the above Prerequisites are met, clone repository and use the Makefile:
   ```
-  go get -u github.com/nexenta/nexenta-docker-driver/...
+  git clone https://github.com/nexenta/nexenta-docker-driver
+  make
```

Comment:
Did you test it? You removed section 'all' from Makefile, so bare 'make' may not work.